packages/manager/modules/pci/src/projects/project/ssh-keys/add/add.html

```diff
@@ -0,0 +1,18 @@
+<form name="pci_projects_project_sshKeys_add_key" novalidate>
+    <oui-modal
+        data-heading="{{:: 'pci_projects_project_sshKeys_add_title' | translate}}"
+        data-primary-action="$ctrl.add()"
+        data-primary-label="{{:: 'pci_projects_project_sshKeys_add_submit_label' | translate }}"
+        data-primary-disabled="$ctrl.isLoading || pci_projects_project_sshKeys_add_key.$invalid"
+        data-secondary-action="$ctrl.goBack()"
+        data-secondary-label="{{:: 'pci_projects_project_sshKeys_add_cancel_label' | translate }}"
+        data-on-dismiss="$ctrl.goBack()"
+        data-loading="$ctrl.isLoading">
+        <oui-field label="{{ ::'pci_projects_project_sshKeys_add_name' | translate }}" size="xl">
```

Comment:
According to the oui-modal implementation there are some missing `data-` prefixes inside this template.